docs/api/functions/getLangfuseContext.md

```
@@ -0,0 +1,90 @@
+requestId, traceName, and metadata. Returns undefined if no context is set.
+
+## Returns
+
+`LangfuseContext` \| `undefined`
+
+The current LangfuseContext or undefined if no context is set
+
+### LangfuseContext Properties
+
+| Property | Type | Description |
+|----------|------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)